canonical-websites/www.ubuntu.com
# 2381 ues takeover

## Done

Add new UES takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Make sure the takeover correlates to the [copy doc](https://docs.google.com/document/d/1yjUvj-B_7KeAvV3HlNnnq45Ndc1SuK8Q1bwOZcgBVUs/edit) and [design](https://app.zenhub.com/workspace/o/canonical-websites/www.ubuntu.com/issues/2381) 


## Issue / Card

Fixes #2381
